src/tester.tsx

```diff
@@ -6,7 +6,10 @@ import {
   sleep,
 } from './utils';
 
-const NullComponent = (props) => <Fragment {...props} />;
+import ConfigurationClass from './ConfigurationClass';
+import { IHook, IProfile, ITesterOps, IWrapper, ComponentClass } from './interfaces';
+
+const NullComponent: React.FC<any>  = (..._props: any[]) => (<Fragment />);
```

Comment:
I think you forgot to apply the props to the component ? 🤔

Reply:
It was causing errors and I kinda don't get the point of applying props to nothing. Could you explain?

Reply:
Nevermind, figured it out!

Reply:
Me neither tbh since I don't fully remember why I was using this (probably custom rendered JSX).

But my point was just about making it exactly like it was before.

I'm glad you got it working ! 👍